src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
Is a new stat completely out of scope for this PR? 
If so, buffing dex further is probably not necessary.